avocado/utils/ssh.py

```diff
@@ -155,11 +156,17 @@ def cmd(self, command):
 
         :param command: the command to execute over the SSH session
         :param command: str
+        :param command: when True run subprocess in the background
+        :param bg_processes: bool
```

Comment:
Here it should also be `:type bg_processes: bool`, if you keep the same parameter name.